validphys2/src/validphys/datafiles/new_commondata/LHCB_Z0_13TEV_2022/uncertainties_dimuon_y.yaml

```diff
@@ -0,0 +1,897 @@
+definitions:
+  Statistical_uncertainty:
+    description: Total (correlated) statistical uncertainty with correlation matrix.
+    treatment: ADD
+    type: CORR
+  Luminosity:
+    description: Systematic uncertainty from the luminosity.
+    treatment: ADD
+    type: LHCB_LUM
+  Efficiency:
+    description: Correlated uncertainties from the selection efficiencies with correlation
+      matrix.
+    treatment: MULT
+    type: CORR
+  Background_uncorrelated:
+    description: Background contributions from heavy flavours, misidentified hadrons
+      and physics processes.(uncorrelated)
+    treatment: MULT
+    type: UNCORR
+  Background_correlated:
+    description: Background contributions from heavy flavours, misidentified hadrons
+      and physics processes.(correlated)
+    treatment: MULT
+    type: CORR
+  Final_state_radiation_uncorrelated:
+    description: Correlated uncertainties from final state radiation corrections.(uncorrelated)
+    treatment: MULT
+    type: UNCORR
+  Final_state_radiation_correlated:
+    description: Correlated uncertainties from final state radiation corrections.(correlated)
+    treatment: MULT
+    type: CORR
+  Closure_uncorrelated:
+    description: Correlated uncertainties from closure test.(uncorrelated)
+    treatment: MULT
+    type: UNCORR
+  Closure_correlated:
+    description: Correlated uncertainties from closure test.(correlated)
+    treatment: MULT
+    type: CORR
+  Alignment_uncorrelated:
+    description: Correlated uncertainties from detector alignment and momentum scale
+      calibration.(uncorrelated)
+    treatment: MULT
+    type: UNCORR
+  Alignment_correlated:
+    description: Correlated uncertainties from detector alignment and momentum scale
+      calibration.(correlated)
+    treatment: MULT
+    type: CORR
+bins:
+- Statistical_uncertainty:
```

Comment:
I have a feeling that I misunderstood how uncertainties must be listed in the file. Could please check if they are implemented correctly, @scarlehoff ?

Reply:
The uncertainties need to be listed one per bin:

```yaml
bins:
- stats: 0.10
   some_name: 11.0
   some_other_name: 143
- stats: 0.20 
   some_name: 143
```

and so on.
The statistical uncertainty should always be named `stat`.